src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+import java.util.Objects;
+
+public class Block {
+    private String value;
+
+    public Block(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Please check how to write single line conditionals according to the coding standards.